### ISSUE TITLE
FTBFS 1.2.0

### ISSUE THREAD

**Issue by jengelh** [iptraf-ng/iptraf-ng#25]
```
16:54 a4:../iptraf-ng/iptraf-ng » make CFLAGS=-O3 -j16
    CC src/ipfilter.o
    CC src/fltmgr.o
    CC src/ipfrag.o
    CC src/servname.o
    CC src/timer.o
    CC src/serv.o
    CC src/revname.o
    CC src/pktsize.o
    CC src/landesc.o
    CC src/options.o
    CC src/promisc.o
    CC src/ifaces.o
    CC src/usage.o
    CC src/iptraf.o
    CC src/itrafmon.o
    CC src/wrapper.o
    CC src/parse-options.o
    CC src/packet.o
    CC src/tcptable.o
    CC src/othptab.o
    CC src/ifstats.o
    CC src/detstats.o
    CC src/deskman.o
    CC src/hostmon.o
    CC src/fltedit.o
    CC src/cidr.o
    CC src/counters.o
    CC src/rate.o
    CC src/capture-pkt.o
    CC src/sockaddr.o
    CC src/capt.o
    CC src/capt-recvmsg.o
    CC src/capt-recvmmsg.o
    CC src/capt-mmap-v2.o
    CC src/capt-mmap-v3.o
    CC src/rvnamed.o
src/capt-recvmmsg.c: In function ‘capt_recvmmsg_find_filled_slot’:
src/capt-recvmmsg.c:27:19: error: invalid use of undefined type ‘struct mmsghdr’
   27 |   if (data->msgvec[slot].msg_len != 0)
```

**Comment by pecameron** [comment]
@jengelh  Try: 
make CFLAGS="-O3 -std=gnu99 -D_GNU_SOURCE" -j16

**Comment by jengelh** [comment]
That succeeds, but that wasn't the point. The point is that mandatory cflags are stuffed in the wrong variable, a problem that wasn't there when Makefile.am was around.

**Comment by vsamel** [comment]
Fixed in #27.
